priv/rest.erl.eex

```diff
@@ -103,9 +103,12 @@
     {Query_, Input} = aws_request:build_headers(QueryMapping, Input2),<% else %>
     Query_ = [],
     Input = Input2,
-<% end %><%= if length(action.response_header_parameters) > 0 do %>
+<% end %><%= if length(action.response_header_parameters) > 0 do %><%= if AWS.CodeGen.RestService.Action.method(action) == "head" do %>
     case request(Client, Method, Path, Query_, CustomHeaders ++ Headers, Input, Options, SuccessStatusCode) of
```

Comment:
This line is identical to 110, right? 

Is there a reason why the `if` branching is not done without including this line inside each branch?

Reply:
No reason now that I think about it 👍 Will fix